ocdsdata/base.py

```diff
@@ -169,6 +176,8 @@ def run_fetch(self):
 
         data = self.metadata_db.get_next_filestatus_to_fetch()
         while data:
+
```

Comment:
Could the "Starting run_X for file X" messages be debug level? Otherwise, the log becomes very verbose, and will crowd out any more significant messages we add later.

Reply:
Guess so ... I think I felt this should be a level above some other stuff that should be debug. But we are maybe starting to guess a bit to much with to little data .... 

Certainly I'm happy to come back to this and potentially change it as part of a full discussion and review when adding lots more messages later - or happy to change now if that's the feeling.

Reply:
Sure, we can discuss appropriate levels as part of #139.